packages/http-client-js/src/components/http-request.tsx

```diff
@@ -1,56 +1,56 @@
-import { Children, code, refkey, Refkey, StatementList } from "@alloy-js/core";
+import { Children, code, List, refkey, Refkey, StatementList } from "@alloy-js/core";
 import * as ts from "@alloy-js/typescript";
 import { Reference } from "@alloy-js/typescript";
-import { ClientOperation } from "@typespec/http-client";
+import { HttpOperation } from "@typespec/http";
 import { EncodingProvider } from "./encoding-provider.jsx";
 import { uriTemplateLib } from "./external-packages/uri-template.js";
 import { HttpRequestOptions } from "./http-request-options.js";
 import { HttpRequestParametersExpression } from "./http-request-parameters-expression.js";
 import { getOperationOptionsParameterRefkey } from "./operation-parameters.jsx";
 
 export interface HttpRequestProps {
-  operation: ClientOperation;
+  httpOperation: HttpOperation;
   responseRefkey?: Refkey;
 }
 
 export function HttpRequest(props: HttpRequestProps) {
   const operationUrlRefkey = refkey();
   const requestOptionsRefkey = refkey();
   const httpResponseRefkey = props.responseRefkey ?? refkey();
-  const verb = props.operation.httpOperation.verb;
+  const verb = props.httpOperation.verb;
   return (
-    <StatementList>
-      <HttpRequest.Url operation={props.operation} refkey={operationUrlRefkey} />
+    <List>
+      <StatementList>
+        <HttpRequest.Url httpOperation={props.httpOperation} refkey={operationUrlRefkey} />
 
-      <HttpRequestOptions operation={props.operation} refkey={requestOptionsRefkey} />
+        <HttpRequestOptions httpOperation={props.httpOperation} refkey={requestOptionsRefkey} />
 
-      <ts.VarDeclaration name="response" refkey={httpResponseRefkey}>
-        {code`
+        <ts.VarDeclaration name="response" refkey={httpResponseRefkey}>
+          {code`
       await client.pathUnchecked(${(<Reference refkey={operationUrlRefkey} />)}).${verb}(${(<Reference refkey={requestOptionsRefkey} />)})
-      
+      `}
+        </ts.VarDeclaration>
+      </StatementList>
```

Comment:
can you add <hbr/> after </StatementList> to keep the line break and reduce the changes in the tests?

Reply:
done, but pls note this could work in `StatementList` not after.